content/docs/get-started-with-neon/production-readiness.md

```diff
@@ -4,7 +4,7 @@ subtitle: Neon features for real-world workloads
 enableTableOfContents: true
 ---
 
-Learn how autoscaling, autosuspend, bottomless storage, change data capture, and instant-read replicas can improve performance, reliability, and efficiency for your production environments.
+Learn how autoscaling, scale-to-zero, Neon's purpose-built storage architecture, change data capture, read replicas, and support for thousands of connections can improve performance, reliability, and efficiency for your production environments.
```

Comment:
"purpose-built" feels a bit vague. What about "Neon's separated storage architecture" or "Neon's disaggregated storage architecture"?

Reply:
Not sure either of those is better, but I did pause on purpose-built. "cloud-native" is maybe another option

Reply:
Thank you. Dropped "purpose-built"